_includes/header.html

```diff
@@ -17,8 +16,10 @@
     <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.2.1/js/bootstrap.min.js"
         integrity="sha384-B0UglyR+jN6CkvvICOB2joaf5I4l3gm9GU6Hc1og6Ls7i6U/mkkaduKaBhlAXv9k"
         crossorigin="anonymous"></script>
-    <script src="https://jsxss.com/assets/xss.js"
-        crossorigin="anonymous"></script>
+    <script src="https://jsxss.com/assets/xss.js" crossorigin="anonymous"></script>
+
+    <!-- Including Leaflet API -->
+    <script src="http://cdn.leafletjs.com/leaflet-0.7.3/leaflet.js"></script>
```

Comment:
Some browsers may take issue with this src tag. They will throw "Mixed Content" errors if you try to access an HTTP resource from an HTTPS website. It's generally best to use HTTPS over HTTP wherever possible.

For example, here are Chrome's logs:

![image](https://user-images.githubusercontent.com/10343470/141164709-9e0cfebe-4f4c-4378-871e-fab75ae1c080.png)